index.html

```diff
@@ -0,0 +1,141 @@
+          </button>
+         </div>
+
+          <div class="project-4">
+      <img src="images/Project 4.png" alt="" class="project4">
+      <div class="work-4">
+ <h2 class="forth-project">Project name goes here</h2>
+<div class="languages-4">
+          <button type="button" class="mark-up-4">HTML/CSS</button>
+          <button type="button" class="react-4">Ruby on Rails</button>
+          <button type="button" class="responsive-4">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-4">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+<div class="project-5">
+      <img src="images/Project 5.png" alt="" class="project5">
+      <div class="work-5">
+ <h2 class="fifth-project">Project name goes here</h2>
+<div class="languages-5">
+          <button type="button" class="mark-up-5">HTML/CSS</button>
+          <button type="button" class="react-5">Ruby on Rails</button>
+          <button type="button" class="responsive-5">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-5">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+    <div class="project-6">
+      <img src="images/Project 6.png" alt="" class="project6">
+      <div class="work-6">
+ <h2 class="sixth-project">Project name goes here</h2>
+<div class="languages-6">
+          <button type="button" class="mark-up-6">HTML/CSS</button>
+          <button type="button" class="react-6">Ruby on Rails</button>
+          <button type="button" class="responsive-6">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-6">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>     
+    </section>
+
+    <section class="about-me">
+      <h1 class="me">About me</h1>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    <button type="button" class="cv"><h6 class="reference">Get my CV</h6><img src="images/ic_download.png" alt=""></button>
+    <div class="lang">
+      <strong>Languages</strong> 
+      <ul class="part-1">
+        <li>JavaScript (ES5/ES6)</li>
+        <li>Ruby</li>
+        <li>HTML</li>
+        <li>CSS</li>
+      </ul>
+    </div>
+
+    <div class="fm-works">
+      <strong class="frameworks">Frameworks</strong>
+      <ul class="part-2">
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>RSpec</li>
+        <li>Capibara</li>
+        <li>Selenium</li>
+      </ul>
+    </div>
+
+    <div class="talent">
+      <strong class="skills">Skills</strong>
+      <ul class="part-3">
+        <li>Codekit</li>
+        <li>Codepen</li>
+        <li>GitHub</li>
+        <li>GitLab</li>
+        <li>Terminal</li>
+      </ul>
+    </div>
```

Comment:
- [ OPTIONAL ] The technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in a `ul` element and each of them in `li` element. Kindly consider changing that.